frontend/src/component/feature/CreateFeature/CreateFeature.tsx

```diff
@@ -137,6 +137,7 @@ const CreateFeature = () => {
                 handleCancel={handleCancel}
                 mode="Create"
                 clearErrors={clearErrors}
+                featureNaming={projectInfo.featureNaming}
```

Comment:
Will this always exists? According to the type in featureForm it can be undefined.

Reply:
Yes. So we send it down. If we have it, all good, otherwise featureForm handles that too. Am I missing something that makes this not work in some cases?